dashboards/lodestar_vm_host.json

```diff
@@ -307,7 +307,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "(nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"} + discv5_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"} + network_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"} + lodestar_historical_state_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"})\nor\n(nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"} + discv5_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"} + lodestar_historical_state_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"})",
+          "expr": "(sum(nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"}) + sum(discv5_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"}) + sum(network_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"}) + sum(lodestar_historical_state_worker_nodejs_external_memory_bytes{job=~\"$beacon_job|beacon\"} or vector(0)))\nor\nsum(nodejs_external_memory_bytes{job=~\"$validator_job|validator\"})",
```

Comment:
it seems like we repeated the same metrics twice here which does not make much sense, I think the intend was to have either beacon or validator metrics displayed